docs/customization.md

```diff
@@ -187,13 +187,21 @@ The default is to generate *everything* supported by the specific library. Once
 
 To control the specific files being generated, you can pass a CSV list of what you want:
 ```sh
+# generate the User and Pet APIs only
+--global-property apis="User,Pet"
```

Comment:
looks like depending on the generators, e.g. for python

```
java -jar modules/openapi-generator-cli/target/openapi-generator-cli.jar generate -g python -i modules/openapi-generator/src/test/resources/3_0/petstore.yaml -o /tmp/python --global-property apis="user:pet"
```

(note that colon is used as a separator)

Reply:
Nice catch, it is actually a colon for all generators. The `--global-property` option accepts a comma separated list of key value pairs, so none of the key/value pairs can have a comma in them. After the split on the comma is done to separate out each key/value pair, CodegenConfigurator replaces the colon with a comma:
```
            configurator.addGlobalProperty(entry.getKey(), entry.getValue().replace(":", ","));
```

I was using a yaml config file with my actual schema I was generating at the time, so I never noticed I had it wrong since yaml doesn't have that issue.

At any rate, updated now to use the colon instead of the comma.